pandas/tests/scalar/timedelta/test_constructors.py

```diff
@@ -204,15 +204,26 @@ def test_overflow_on_construction():
         Timedelta(timedelta(days=13 * 19999))
 
 
-def test_construction_out_of_bounds_td64():
```

Comment:
This test was already here and should have caught #36615.
I parametrized it, but this is still a bit puzzling